src/common/debugAdapter.ts

```diff
@@ -302,8 +310,24 @@ export class DebugAdapter implements vscode.DebugAdapter {
 		return normalized;
 	}
 
-	private _translateFromWorkspacePath(workspacePath: string): string {
-		const normalized = workspacePath.replace(/\\/g, '/');
+	private _convertToUriString(path: string): string {
+		if (path.includes('://')) {
+			return path;
+		} else {
+			try {
+				const output = vscode.Uri.file(path);
+				return output.toString();
+			} catch {
+				return path;
+			}
+		}
+	}
+
+	private async _translateFromWorkspacePath(workspacePath: string): Promise<string> {
+		// Have to wait for path mappings to come in from the device
+		await this._pathMappingsComplete;
+
+		const normalized = this._convertToUriString(workspacePath.replace(/\\/g, '/'));
```

Comment:
Also had to fix this for the non remote case. Local files will not be of the URI form. Maybe that could be fixed elsewhere, but did this for now.

Reply:
I pinged Andre to find out if this is on purpose (some legacy) and if we need to deal with this on out end. Will keep you posted